odoo_backup_sh/controllers/main.py

```diff
@@ -236,6 +236,14 @@ def fetch_dashboard_data(self):
         })
         return dashboard_data
 
+    @http.route('/odoo_backup_sh/open_iap_recharge', type='http', auth="user")
+    def open_iap_recharge(self, **kwargs):
+        data = {'params': {'user_key': BackupController.get_config_values(
```

Comment:
Doesn't ``self.get_config_values`` work?

Reply:
It was copy-pasted from models. Changed.